README.md

```diff
@@ -3,35 +3,97 @@
 </br>
 </br>
 
-<div align="center">
+<p align="center">
+  <img src="assets/logo.png" alt="xk6-disruptor" width="300"></a>
+  <br>
+  "Like Unit Testing, but for <strong>Reliability</strong>"
+  <br>
+</p>
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
 
+xk6-disruptor is an extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
 
-</div>
+<blockquote align="center">
```

Comment:
Here? Or did I misunderstand what you meant? :D @pablochacin

Reply:
Yes, only that not centered.